volume/store/store.go

```diff
@@ -146,7 +146,7 @@ func (s *VolumeStore) Purge(name string) {
 	v, exists := s.names[name]
 	if exists {
 		if _, err := volumedrivers.ReleaseDriver(v.DriverName()); err != nil {
-			logrus.Errorf("Error dereferencing volume driver: %v", err)
+			logrus.Errorf("Error removing volume driver: %v", err)
```

Comment:
how about `Error releasing volume driver`?

Reply:
Yes, "releasing" would be better; perhaps use the same as is used in `VolumesStore.create()`; https://github.com/moby/moby/blob/4d8598ad0506b29c12632c1b8ed92eb58fc2f0e2/volume/store/store.go#L412-L414

Reply:
ok , thanks ,I just have a try  the commit process ,but  everyone is so enthusiastic, I will  work harder